fabric-common-tags-api-v1/src/datagen/java/net/fabricmc/fabric/impl/tag/common/datagen/generators/EnchantmentTagGenerator.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.tag.common.datagen.generators;
+
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.enchantment.Enchantments;
+import net.minecraft.util.registry.Registry;
+
+import net.fabricmc.fabric.api.datagen.v1.FabricDataGenerator;
+import net.fabricmc.fabric.api.datagen.v1.provider.FabricTagProvider;
+import net.fabricmc.fabric.api.tag.v1.CommonEnchantmentTags;
+
+public class EnchantmentTagGenerator extends FabricTagProvider<Enchantment> {
+	public EnchantmentTagGenerator(FabricDataGenerator dataGenerator) {
+		super(dataGenerator, Registry.ENCHANTMENT, "Enchantment Tags");
+	}
+
+	@Override
+	protected void generateTags() {
+		getOrCreateTagBuilder(CommonEnchantmentTags.INCREASES_BLOCK_DROPS)
+				.add(Enchantments.FORTUNE);
+		getOrCreateTagBuilder(CommonEnchantmentTags.INCREASES_ENTITY_DROPS)
+				.add(Enchantments.LOOTING);
```

Comment:
Would luck of the sea go here somewhere?

Reply:
Fishing is a bit of a different mechanic to mining or fighting, so I don't think it fits within either of those tags